tool/lldb_tool.py

```diff
@@ -12,8 +12,109 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """A tool for LLM agents to interact within a LLDB."""
+import logging
+import os
+import subprocess as sp
+
+from experiment import oss_fuzz_checkout
+from experiment.benchmark import Benchmark
+from results import RunResult
 from tool.base_tool import BaseTool
 
+logger = logging.getLogger(__name__)
+
+# The directory in the oss-fuzz image
+JCC_DIR = '/usr/local/bin'
+
 
 class LLDBTool(BaseTool):
-  pass
+  """A tool for LLM agents to interact within a LLDB."""
+
+  def __init__(self, benchmark: Benchmark, name: str, project: str,
+               result: RunResult) -> None:
+    super().__init__(benchmark, name)
+    self.project = project
+    self.result = result
+    self.image_name = self._prepare_project_image()
+    self.container_id = self._start_docker_container()
+
+  def _prepare_project_image(self) -> str:
+    """Prepares the project's OSS-Fuzz docker image and returns the image name.
+    """
+    image_name = f'gcr.io/oss-fuzz/{self.project}'
```

Comment:
Would it be better to use `gcr.io/oss-fuzz/{self.project}-lldb` to distinguish it from the normal image without `lldb`?